source/plugin/Assets/GoogleMobileAds/Platforms/Unity/BannerClient

```diff
@@ -0,0 +1,235 @@
+// Copyright (C) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System;
+using System.Reflection;
+using GoogleMobileAds.Api;
+using GoogleMobileAds.Common;
+
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace GoogleMobileAds.Unity
+{
+    public class BannerClient : DummyUtils, IBannerClient
```

Comment:
why not extend AdBehavior?

Reply:
There's some complications with deriving the class from a class that is using MonoBehaviour. I wouldn't be able to have code outside of functions.

Reply:
ok that's fine then.